Sources/CryptorRSA/CryptorRSA.swift

```diff
@@ -222,7 +222,9 @@ public class CryptorRSA {
 			}
 			
 			#if os(Linux)
-                
+                if algorithm == .gcm {
```

Comment:
This special-casing of GCM seems a bit hacky... Would it be better to `switch algorithm { }` and call a function depending on the result?  In other words, have `encryptedSHA(with: key)` as well?

Reply:
The switch would be:
```swift
switch algorithm {
    case .gcm:
         return try encryptedGCM(with: key)
    default:
         return try encryptedSHA(with: key)
}
```
Which seems a bit strange for a switch statement. 
I could make the SHA a separate function and do an if else statement. 

Alternatively we could just make encryptedGCM public as it's own function.

Reply:
I would list all the SHA cases explicitly rather than using `default:` but yes this is what I'm suggesting. This function is called `encrypted()` - now that it does more than just SHA I don't see why all the SHA code belongs in it?